app/javascript/dashboard/routes/dashboard/settings/profile/Index.vue

```diff
@@ -85,27 +91,25 @@
               {{ $t('PROFILE_SETTINGS.FORM.PASSWORD_CONFIRMATION.ERROR') }}
             </span>
           </label>
+          <woot-submit-button
```

Comment:
Not a blocking suggestion, We have introduced a new button component `woot-button` @ `app/javascript/dashboard/components/ui/WootButton.vue` Can try if it's a good fit here, with the `type="submit"` attribute ?

Reply:
Sure